### ISSUE TITLE
mini batch defined for training?

### ISSUE THREAD

**Issue by tingsun123** [jayavardhanr/End-to-end-Sequence-Labeling-via-Bi-directional-LSTM-CNNs-CRF-Tutorial#3]
Great implementation from the paper!

The paper used mini-batch gradient descent with batch_size of 10. But I can't seem to find it in the training step. It seems that you are trying 1 obs at a time.. Can you confirm?
If so, do you plan to update it? 

Thanks!

**Comment by jayavardhanr** [comment]
Yes, the current code performs 1 obs at a time. I will make the mini-batch change sometime soon.
Thanks!

**Comment by tingsun123** [comment]
Cool. Thanks!
